lib/samson/integration.rb

```diff
@@ -0,0 +1,16 @@
+module Samson
+  class Integration
+
+    @@integrations = Rails.root.join('app','controllers', 'integrations').children(false).map do |controller_path|
```

Comment:
It doesn't seem idiomatic to have a constant that we expect to mutate if plugins want to register themselves here